### PR TITLE
Create input component

### PR DESCRIPTION
**Pull request by CBinh710**
# Result
image

### CODE REVIEW

File: practice-1/src/styles/components/input.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+.input-sm {
+  font-size: var(--font-size-sm);
+  padding: 5px 0;
+}
+
+.input-none-border {
+  border: none;
+}
+
+.image-container {
```
Comment (phuocduong-agilityio) [inline practice-1/src/styles/components/input.css]: This naming is not related to input?

Comment (CBinh710) [reply practice-1/src/styles/components/input.css]: I fixed all, thanks for your comment
